babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -1820,10 +1820,22 @@ SqlNode SqlSelectTopN(SqlParserPos pos) :
 
 SqlHostVariable SqlHostVariable() :
 {
+    String name;
+    SqlParserPos pos;
 }
 {
     <COLON>
-    <IDENTIFIER> {
-        return new SqlHostVariable(unquotedIdentifier(), getPos());
-    }
+    (
+        <IDENTIFIER>
+        {
+            name = unquotedIdentifier();
+            pos = getPos();
```

Comment:
nit: Since this is present in both of the cases, can it be pulled out and get set after the parentheses?

Reply:
Yeah, we could get rid of the temp variable too and just call it in the constructor